root/documentation/vep.conf

```diff
@@ -295,7 +345,12 @@
         type=Boolean
         description=Include HGVS nomenclature based on Ensembl stable identifiers
         default=0
-      </hgvs> 
+      </hgvs>
+      <ambiguous_hgvs>
```

Comment:
This is not relevant for the region endpoint as it takes genomic locations+ alleles as input, not HGVS